lib/tasks/review_profile.rake

```diff
@@ -31,40 +41,46 @@ namespace :users do
             disavowal_token: disavowal_token,
           )
 
-          puts "User's profile has been activated and the user has been emailed."
+          STDOUT.puts "User's profile has been activated and the user has been emailed."
         else
-          puts "There was an error activating the user's profile please try again"
+          STDOUT.puts "There was an error activating the user's profile please try again"
```

Comment:
```suggestion
          STDOUT.puts "There was an error activating the user's profile. Please try again"
```
Might as well fix the grammar while we're here.

Is specifying STDOUT for purposes of making tests easier to write?

Reply:
Great catch, thanks.

And yes, `STDOUT` was for stubbing so that:
1. we can check what's printed
2. we can run the test and not see the messages printed to the console between spec lines